src/main/java/com/exadel/project/internship/controller/InternshipController.java

```diff
@@ -19,12 +21,12 @@
     private InternshipService internshipService;
 
     @GetMapping("internship")
-    public List<Internship> findAllInternships(@RequestParam(value = "search", required = false) String search){
-        return internshipService.getAllEntities(search);
+    public List<InternshipDTO> findAllInternships(@RequestParam(value = "search", required = false) String search){
+        return internshipService.getAllDTOs(search);
```

Comment:
pls rename method in service into `getAll`

Reply:
done